region/crateria/central/Crateria Power Bomb Room.json

```diff
@@ -86,24 +86,37 @@
       "link": [1, 1],
       "name": "Alcoon Farm",
       "requires": [
-        {"or": [
-          "canDodgeWhileShooting",
-          "ScrewAttack",
-          "Wave",
-          "Ice",
-          "Spazer",
-          "Plasma"
-        ]},
         {"resetRoom": {
           "nodes": [1]
         }},
-        {"refill": ["PowerBomb"]}
+        {"or": [
+          {"and": [
+            "ScrewAttack",
+            {"cycleFrames": 575}
+          ]},
+          {"and": [
+            "Plasma",
+            {"cycleFrames": 650}
+          ]},
+          {"and": [
+            {"or": [
+              "Wave",
+              "Spazer"
+            ]},
+            {"cycleFrames": 1000}
+          ]},
+          {"and": [
+            {"or": [
+              "canDodgeWhileShooting",
+              "Ice"
+            ]},
+            {"cycleFrames": 1210}
+          ]}
+        ]}
       ],
+      "farmCycleDrops": [{"enemy": "Alcoon", "count": 3}],
```

Comment:
This is a farm you would take on no items if you needed PBs.
And is very fast with missiles.

Reply:
Added a Missile option now. Currently the randomizer wouldn't use it, because it's not set up to trade off ammo types like that, but that's something that can be refined, and in any case it belongs as an option in the logic data.